src/main.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+class Todo  {
+    constructor() {
```

Comment:
use 2 whitespaces instead of 4
https://github.com/mate-academy/style-guides/blob/master/javascript.md#whitespace--spaces